pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -515,23 +514,20 @@ def _validate_monitor_key(self, trainer):
             )
             raise MisconfigurationException(m)
 
-    def _get_metric_interpolated_filepath_name(self, ckpt_name_metrics: Dict[str, Any], epoch: int, step: int):
-        filepath = self.format_checkpoint_name(epoch, step, ckpt_name_metrics)
-        version_cnt = 0
+    def _get_metric_interpolated_filepath_name(self, monitor_candidates: Dict[str, Any], epoch: int, step: int) -> str:
+        filepath = self.format_checkpoint_name(epoch, step, monitor_candidates)
+        version = 1
```

Comment:
Might be better to do: `version = self.STARTING_VERSION`

Reply:
Done